src/data/markdown/docs/30 k6-browser/02 API/03 BrowserType/connect--options--.md

```diff
@@ -0,0 +1,49 @@
+---
+title: 'connect(wsURL, [options])'
+excerpt: 'Browser module: BrowserType.connect method'
+---
+
+Connects to an existing browser instance.
+
+| Parameter         | Type     | Default | Description                                                                                                                                                                                                                                                           |
+|-------------------|----------|---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| debug             | boolean  | `false` | All CDP messages and internal fine grained logs will be logged if set to `true`.                                                                                                                                                                                      |
+| slowMo            | string   | `null`  | Slow down input actions and navigation by the specified time e.g. `'500ms'`.                                                                                                                                                                                          |
+| timeout           | string   | `'30s'` | Default timeout to use for various actions and navigation.                                                                                                                                                                                                            |
+
+
+### Returns
+
+| Type   | Description                                            |
+|--------|--------------------------------------------------------|
+| object | [Browser](/javascript-api/k6-browser/api/browser/) object |
+
+
+## Example
+
+<CodeGroup labels={[]}>
+
+```javascript
+import { chromium } from 'k6/experimental/browser';
+
+export default async function () {
+  const wsURL = 'ws://browser.farm.example.com:9222/devtools/browser/a7ee4f2d-35cf-4478-a333-f597e1532ab0';
```

Comment:
I think from a user's perspective `127.0.0.1:9222` looks friendlier.

```suggestion
const wsURL = 'ws://127.0.0.1:9222/devtools/browser/a7ee4f2d-35cf-4478-a333-f597e1532ab0';
```

Reply:
Actually, initially I was gonna define it as `127.0.0.1`, as that probably represents running Chrome locally in your machine. But I thought that it would look more "misleading" maybe and that it was better to define a DNS name that clearly identified that we are targeting some sort of browser or browser farm. But no strong opinion, we can set it to `localhost`.

Reply:
Changed in [b1fa39e](https://github.com/grafana/k6-docs/pull/1114/commits/b1fa39e0cae20adcd24571273da838c9f194ee0e).